src/console_cmds.cpp

```diff
@@ -256,8 +256,8 @@ DEF_CONSOLE_CMD(ConResetTile)
 
 /**
  * Scroll to a tile on the map.
- * @param arg1 tile tile number or tile x coordinate.
- * @param arg2 optionally tile y coordinate.
+ * param x tile number or tile x coordinate.
```

Comment:
I don't think this will generate right.

Reply:
The trouble with the console commands is that they don't actually have "proper" parameters - once the `#define` expands, they have argc & argv, not whatever console commands actually take. Probably need a better way of documenting console commands generally, but I decided that was out of scope for what I was trying to do (fix warnings)

Reply:
I think it might work to use the `@fn` command to introduce the functions, and then you should be allowed to use `@param` as well. But it would still not match up with the C++ function signature if you document the "logical" parameters... yeah, not going to work.